### PR TITLE
[opentracing] Update VMs 2020-12

### PR DESCRIPTION
**Pull request by BillyONeal**

### CODE REVIEW

**Comment by BillyONeal** [comment]
@PhoebeHui @JonLiu1993 No point reviewing right now because I'm still working on making it actually work :)

**Comment by PhoebeHui** [comment]
Sure, list additional packages required in VM.

PR | Platform | Package
-- | -- | --
14547  14545 | osx | nasm
14228 | osx | gperf
14774 | linux| libasound2-dev
14412 | all | open ftp port 21 or alternate host site

**Comment by BillyONeal** [comment]
@PhoebeHui osx isn't in Azure so it won't be part of this update, but it'll be right after. 14412 is already here. I'll take note to look to address 14774

**Comment by JackBoosY** [comment]
We need install xcode and vulkan on our osx pipeline machine too. See https://github.com/microsoft/vcpkg/pull/14819.

**Comment by NancyLi1013** [comment]
Please help install `libxcb-res0-dev` on Linux platform, which is required in #13467.

File: scripts/azure-pipelines/windows/provision-image.txt [hunk 1]
```diff
@@ -68,7 +69,7 @@ if ([string]::IsNullOrEmpty($AdminUserPassword)) {
   Write-Host 'AdminUser password supplied; switching to AdminUser.'
   $PsExecPath = Get-TempFilePath -Extension 'exe'
   Write-Host "Downloading psexec to: $PsExecPath"
-  & curl.exe -L -o $PsExecPath -s -S https://live.sysinternals.com/PsExec64.exe
+  & curl.exe -L -o $PsExecPath -s -S http://live.sysinternals.com/PsExec64.exe
```
Comment (strega-nil) [inline scripts/azure-pipelines/windows/provision-image.txt:72]: This seems... bad. Why are we switching from SSL to non-SSL?

**Review by strega-nil — APPROVED** [review]

File: scripts/azure-pipelines/windows/provision-image.txt (hunk 1 above)
Comment (BillyONeal) [reply scripts/azure-pipelines/windows/provision-image.txt:72]: The SSL endpoint is gone :(. I agree it is bad.

**Pull request merged**